purchase_discount/tests/test_product_supplierinfo_discount.py

```diff
@@ -47,32 +47,38 @@ def setUpClass(cls):
         )
 
     def test_001_purchase_order_partner_3_qty_1(self):
-        self.po_line_1._onchange_quantity()
-        self.assertEqual(
-            self.po_line_1.discount,
-            10,
-            "Incorrect discount for product 6 with partner 3 and qty 1: "
-            "Should be 10%",
-        )
+        purchase_form = Form(self.purchase_order)
+        with purchase_form.order_line.edit(0) as line:
+            line.product_qty = 1.0
+
```

Comment:
Don't add empty lines inside of a method

Reply:
done